docs/images/siddhi-logo.svg

```diff
@@ -0,0 +1,19 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Same. Fix in all places applicable.

Reply:
Fixed with commit: `Fix some suggested issues in PR#1.` 8a89f22